src/edge.hal-interface/src/hal_interface.py

```diff
@@ -476,14 +476,14 @@ def __init__(self, abs_config_path):
             logging.getLogger().setLevel(log_level)
 
     async def start(self):
-        await self.init_message_client()
+        await self.init_protocol_gateway()
 
         # Starting all observers
         for bus_config in self.config['can']:
             if bus_config['interface'] == 'mock':
-                bus_observer = MockBusObserver(bus_config, self.client)
+                bus_observer = MockBusObserver(bus_config, self.protocol_gateway)
             else:
-                bus_observer = BusObserver(bus_config, self.client)
+                bus_observer = BusObserver(bus_config, self.broker)
```

Comment:
`self.broker` needs to be replaced by `self.protocol_gateway`

Reply:
Updated.